### ISSUE TITLE
A problem or bug in pytorch 0.4.0a0+063946d ?

### ISSUE THREAD

**Issue by lxtGH** [pytorch/pytorch#6129]
I update my pytorch version to new 0.4 today, but I found my code cann't  run 
My enviroment: 
- OS: Ubuntu16.04.3 LTS
- PyTorch version: 0.4.0a0+063946d
- How you installed PyTorch (conda, pip, source): source
- Python version: 3.6
- CUDA/cuDNN version: 9.0
- GCC version (if compiling from source):gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.9)
run the code like this:

import numpy as np
import torch
from torch.autograd import Variable
shape = (512, 512)
out = np.ones(shape=(shape[0], shape[1]), dtype=np.float32) * 250
out = torch.from_numpy(out).long()
out = Variable(out)
mask = out > 0
**print(mask.data.sum())**

this result got zero ?   the sum() function work wrongly? or the new API change? help me !

**Comment by colesbury** [comment]
Use `mask.long().sum()` for now.

Currently, sum() on a ByteTensor wraps around at 256. We are going to change this before the 0.4 release.

See https://github.com/pytorch/pytorch/issues/5966

**Comment by colesbury** [comment]
(Closing this because we're tracking the problem in the issue above)

**Comment by lxtGH** [comment]
@colesbury Thank you for your reply !!!!
